src/components/Collapsible.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+
+const Collapsible = ({ children, contentDescriptor, commentCount })=> {
+
+    const [isHidden, setIsHidden] = useState(false);
+
+    const toggleIsHidden = ()=> {
+        setIsHidden(!isHidden);
+    }
+
+    return (
+        <section>
+            <button onClick={toggleIsHidden}>
+                {isHidden ? 'Show' : 'Hide'} {contentDescriptor} ({commentCount})
```

Comment:
Nice idea adding the commentCount, if I go to articles/37 which has no comments, I get visual feedback saying that there are no comments. Nice touch!